src/Console/ConsoleIo.php

```diff
@@ -266,7 +266,7 @@ public function ask($prompt, $default = null)
      */
     public function outputAs($mode)
```

Comment:
Oh I misread what ConsoleIo::outputAs() did. It might be nice if the methods had the same name as there would be fewer names to remember later on.